src/pages/Home.vue

```diff
@@ -142,6 +145,18 @@ export default {
   methods: {
     ...mapActions("plioItems", ["purgeAllPlios"]),
     ...mapActions("sync", ["startLoading", "stopLoading"]),
+    async sortByNumberOfViewers(sortOrder) {
+      // invoked when the user clicks the sort icon next to "number of viewers" column
+
+      // remove any existing value of "unique_viewers" field. New value will be pushed below
+      this.sortByFields = this.sortByFields.filter((value) => {
+        return !value.includes("unique_viewers");
+      });
+
+      // push the appropriate field name according to the sort order and make the API call
+      this.sortByFields.push(sortOrder == 1 ? "unique_viewers" : "-unique_viewers");
+      await this.fetchPlioIds();
+    },
```

Comment:
why not keep the function as `sortPlios` and take the param to sort by as input (e.g. `unique_viewers`, `-unique_viewers`)? won't have to do the `== 1` check too and keeps it generic.